background.js

```diff
@@ -0,0 +1,16 @@
+// Add action to context menu
+chrome.contextMenus.create({
+    id: "play-action",
+    title: "Splia",
+    contexts: ["selection"],
+  },
+);
+
+// Handler for centext menu
+const onClick = (info, tab) => {
+  if (info.menuItemId == "play-action") {
+    chrome.tabs.sendMessage(tab.id, {action: "play"});
```

Comment:
```suggestion
    chrome.tabs.sendMessage(tab.id, {
      receiver: "content_script",
      command: "play",
    });
```

Reply:
with this change the added listener in content is not necessary

Reply:
Nice